srcCpp/perftest_publisher.cxx

```diff
@@ -123,6 +126,126 @@ int perftest_cpp::Run(int argc, char *argv[])
     }
 }
 
+bool perftest_cpp::set_main_thread_priority()
```

Comment:
I was wondering if these 3 functions could be actually be declared in the .h of the new file and implemented in the .cxx (outside the class, just as functions). What do you think?

Reply:
I followed your recommendation. I have move these three function to perftestThreadPriorities file inside of the ``PerftestThreadPriorities`` class.

Also, the implementation for c++ and c++03 is the same. So the files inside of srcCpp & srcCpp03 are the same files ready to be moved to srcCommon. Edit